README.md

```diff
@@ -7,29 +7,30 @@ This repo contains docker files and scripts to build docker images that are used
 Docker image inventory:
 
 Maintained images (can be updated by CI)
-- Ubuntu 16.04
-- Ubuntu 18.04
-- Ubuntu 20.04
+- Alpine 3.15
+- CentOS 7
```

Comment:
We should move `CentOS 7` to `Legacy images` and put `CentOS 8` here instead, no?

Reply:
The previous comment is not relevant as we'll deprecate CentOS 8 and move back to CentOS 7